### PR TITLE
Policy Playground Layout Updates

### PR DESCRIPTION
**Pull request by pickjasmine**
Changes the playground to match the updated prototypes/the OPA Rego playground.

### CODE REVIEW

File: components/playground/SelectedPolicy.js [hunk 1]
```diff
@@ -14,61 +14,33 @@
...
-          label={"Clear Policy"}
-          onClick={clearPolicy}
+    <>
+      <p className={textStyles.label}>Rego Policy Code</p>
+      {policy ? (
+        <Code
+          code={policy.regoContent}
+          language={"rego"}
+          className={styles.codeContent}
+          data-testId={"regoPolicyCode"}
```
Comment (alexashley) [inline components/playground/SelectedPolicy.js:34]: This was emitting a warning in the console:
>Warning: React does not recognize the `data-testId` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `data-testid` instead. If you accidentally passed it from a parent component, remove it from the DOM element.


```suggestion
          data-testid={"regoPolicyCode"}
```

File: components/playground/SelectedResource.js [hunk 2]
```diff
@@ -14,84 +14,42 @@
...
-          <LabelWithValue
-            label={"Type"}
-            value={resource.type}
-            className={styles.selectionDetails}
+    <>
+      <p className={textStyles.label}>Occurrence Data</p>
+      {resource ? (
+        <Loading loading={loading}>
+          <Code
+            code={JSON.stringify(data?.originals, null, 2)}
```
Comment (alexashley) [inline components/playground/SelectedResource.js:38]: I'm seeing a prop type warning in the console the first time the occurrence json loads:

>Failed prop type: The prop `code` is marked as required in `Code`, but its value is `undefined`.

**Review by alexashley — COMMENTED** [review]
Not sure the best way to describe, but the `resize` corner in the policy editor causes the parent widget to expand or contract. But on the playground, the resize doesn't affect the parent container at all: 

Screen Shot 2021-05-24 at 5 32 16 PM

Also I probably missed it, but is there a way to do trigger another evaluation? The only thing I could see to do was re-select a policy or resource.

**Comment by pickjasmine** [comment]
> Not sure the best way to describe this (or what the right behavior is), but the `resize` corner in the policy editor causes the parent widget to expand or contract. But on the playground, the resize doesn't affect the parent container at all:

I believe this is because the playground is using flex for the layout but the policy creation page is using flexbox. I think it would be better ot be constrained to the parent container, do you agree? I will make a note to update them to match.


> Also I probably missed it, but is there a way to do trigger another evaluation? The only thing I could see to do was re-select a policy or resource.

Yeah, searching for a new policy or resource is the only way right now. I can add back the button to reset the playground or clear the policy/resource, but I figured once the user is able to edit the policy or resource fields then having the current screen would be the cleanest UI. I do have some ideas around adding a copy button to the blocks of code, so it could be a good place to add another button to remove the selected policy/resource

**Comment by alexashley** [comment]
@pickjasmine I agree it makes sense to limit them to the parent container,  and also maybe to set `resize: none` to remove the ability to change the textarea size (since it's already as big as it can get and can only be made smaller).

**Comment by pickjasmine** [comment]
> @pickjasmine I agree it makes sense to limit them to the parent container, and also maybe to set `resize: none` to remove the ability to change the textarea size (since it's already as big as it can get and can only be made smaller).

I will add the `resize: none` for now. I tried to copy the overflow/scrolling behavior that the playground has, but because the way the syntax highlighting is done I will have to spend some time figuring out if that behavior is possible.

**Review by alexashley — APPROVED** [review]
:shipit:

**Pull request merged**